ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ClaimMapper.java

```diff
@@ -18,6 +18,7 @@
 
 @Component
 public class ClaimMapper {
```

Comment:
There is no test for claimmapper. Needs to be written

Reply:
caseMapper test covers tests for this as ccdClaimData is not an object